test/functional/mempool_limit.py

```diff
@@ -143,12 +143,37 @@ def create_package(self, node, utxos, mempool_evicted_utxo, feerate, size=5):
         package_txids.append(child.hashMalFix)
         return (package_hex, package_txids)
 
-    def create_tx_to_be_evicted(self, node, unspent):
+    def create_tx_to_be_evicted(self, node, unspent, feerate=None):
```

Comment:
There doesn't seem to be any tests that specify `feerate` as an argument. Why is it necessary?